bundles/org.eclipse.jface.text/src/org/eclipse/jface/internal/text/codemining/CodeMiningLineContentAnnotation.java

```diff
@@ -216,4 +216,8 @@ public boolean isInVisibleLines() {
 	public final boolean isAfterPosition() {
 		return afterPosition;
 	}
+
+	public List<ICodeMining> getMinings() {
+		return fMinings;
+	}
```

Comment:
As the discussion shows I don't think anyone needs access to this data structure or should access it! This is even an internal class so no one should ever call any method on this outside the framework itself.